### PR TITLE
Enable `no-console` and `no-alert` ESLint rules 

### PR DESCRIPTION
**Pull request by astorije**
We almost slipped in a few `console.log` statements so this should make it safer. Outside of the current occurrences we have, we shouldn't need any more in the code, or at least not without explicitly acknowledging them.

### CODE REVIEW

File: client/js/socket.js [hunk 1]
```diff
@@ -35,8 +35,6 @@ const socket = io({
 		});
 		// Hides the "Send Message" button
 		$("#submit").remove();
-
-		console.error(data);
```
Comment (astorije) [inline client/js/socket.js]: We really shouldn't be using `console.xxx` in the browser...

File: webpack.config.js [hunk 2]
```diff
@@ -79,8 +79,6 @@ if (process.env.NODE_ENV === "production") {
 		sourceMap: true,
 		comments: false
 	}));
-} else {
-	console.log("Building in development mode, bundles will not be minified.");
```
Comment (astorije) [inline webpack.config.js]: I don't believe this is really helpful at this point, might as well not have it.

File: .eslintrc.yml [hunk 3]
```diff
@@ -9,54 +9,54 @@ env:
...
+  semi-style: [error, last]
+  semi: [error, always]
+  space-before-blocks: error
+  space-before-function-paren: [error, never]
+  space-in-parens: [error, never]
+  space-infix-ops: error
+  spaced-comment: [error, always]
+  strict: error
+  template-curly-spacing: error
+  yoda: error
```
Comment (YaManicKill) [inline .eslintrc.yml]: Is "2" equal to "error"?

Comment (xPaw) [reply .eslintrc.yml]: Yes.

**Pull request merged**